### DISCUSSION TITLE
Crtl-Alt-Del keyboard shortcut (eg. Ctrl-Alt-Del like Ms RDP client)

### DISCUSSION THREAD

**Discussion by jcrmatos** [rustdesk/rustdesk#8366] [General]
Hello,

Please consider adding a Crtl-Alt-Del keyboard shortcut (eg. Ctrl-Alt-Del like Ms RDP client).
This way it won't interfere with the local PC and would send the Ctrl-Alt-Del to the remote.

Thanks,

JM

**Comment by svhp1** [comment]
This is similar to issue #1422 which was marked solved,  However, that solution requires Rust D to be installed.  I have similar question as OP when running as a portable app.  From a remote session and facing a host locked screen, I cannot login with the portable Rust D app version.   Since I have no admin rights, only user rights, I run the portable app.   I need a way of sending "ctrl-alt-del" from the portable app to be able to enter my user credentials to join a domain host PC.
